include/mbedtls/config_adjust_legacy_from_psa.h

```diff
@@ -67,20 +67,26 @@
 
 #if defined(PSA_WANT_ALG_HKDF)
 #if !defined(MBEDTLS_PSA_ACCEL_ALG_HKDF)
+#define MBEDTLS_HKDF_C
```

Comment:
No: PSA has its own implementation of HKDF, separate from `hkdf.c`. The existing code here is correct.

Reply:
This has come up before and we really should have put a comment there.

Reply:
ah I was indeed wondering why it was not caught by the PSA crypto tests.

Reply:
I've added some comments instead.